packages/governance/test/unitTests/snapshots/test-committee.js.md

```diff
@@ -0,0 +1,62 @@
+# Snapshot report for `test/unitTests/test-committee.js`
+
+The actual snapshot is saved in `test-committee.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## committee-open question:mixed, with snapshot
+
+> Under "published", the "committees.Economic_Committee" node is delegated to
+> a committee contract, which publishes data as follows.
+> 
+> See also board marshalling conventions (_to appear_).
+> 
+
+    [
+      [
+        'published.committees.Economic_Committee.latestQuestion',
+        {
+          closingRule: {
+            deadline: 1n,
+            timer: {
+              iface: 'Alleged: ManualTimer',
+            },
+          },
+          counterInstance: {
+            iface: 'Alleged: InstanceHandle',
+          },
+          electionType: 'survey',
+          issue: {
+            text: 'why3',
```

Comment:
This lacks detail on parameter change issues. I think developers of dapp-gov, i.e. us, are the most interested consumers of that API, and we already figured it out.